packages/neon-api/src/api/getTokenBalances.ts

```diff
@@ -24,7 +24,9 @@ export async function getTokenBalances(
   const response = await client.invokeScript(script);
   if (response.state === "FAULT") {
     throw new Error(
-      `Invoke exception: ${response.exception ?? "No exception returned."}`
+      response.exception
+        ? `Invoke exception: ${response.exception}}`
+        : "No exception returned."
```

Comment:
```suggestion
        : "No exception message returned."
```

Reply:
Otherwise you'll still end up in a `exception catch()` handler while it says "no exception returned"